docs/notebooks/generate_differentially_private_synthetic_text.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/gretelai/gretel-blueprints/blob/main/sdk_blueprints/Gretel_Advanced_Tabular_Blueprint.ipynb)\n",
+        "\n",
+        "<br>\n",
+        "\n",
+        "<center><a href=https://gretel.ai/><img src=\"https://gretel-public-website.s3.us-west-2.amazonaws.com/assets/brand/gretel_brand_wordmark.svg\" alt=\"Gretel\" width=\"350\"/></a></center>\n",
+        "\n",
+        "<br>\n",
+        "\n",
+        "## Generate Differentially Private Synthetic Text with Gretel GPT\n",
```

Comment:
Question about templating. Do we want to have some standard blurb in our notebooks that point users to the SDK Blueprint notebooks if this is there first time using the Gretel SDK?